lib/adobe/campaign/language.rb

```diff
@@ -90,104 +88,106 @@ def self.normalize(language_code)
         en_qa
         en_sa
         en_sg
-        en_za
         en_tt
-        en_ae
-        en_gb
+        en_za
         en_zw
+        es_es
+        es_ar
+        es_bo
+        es_cl
+        es_co
+        es_cr
+        es_do
+        es_ec
+        es_gt
+        es_hn
+        es_mx
+        es_ni
+        es_pa
+        es_pe
+        es_pr
+        es_py
+        es_sv
+        es_uy
+        es_ve
         et_ee
-        fo_fo
+        eu_es
         fa_ir
         fi_fi
+        fo_fo
+        fr_fr
         fr_be
         fr_ca
-        fr_fr
+        fr_ch
         fr_lu
         fr_mc
-        fr_ch
         gl_es
-        ka_ge
-        de_at
-        de_be
-        de_de
-        de_it
-        de_li
-        de_lu
-        de_ch
-        el_cy
-        el_gr
         gu_in
         he_il
         hi_in
+        hr_hr
         hu_hu
-        is_is
+        hy_am
         id_id
+        is_is
         it_it
         it_ch
         ja_jp
-        kn_in
+        ka_ge
         kk_kz
+        kn_in
         ko_kr
         ky_kz
-        lv_lv
         lt_lt
+        lv_lv
         mk_mk
-        ms_bn
-        ms_my
-        mr_in
         mn_mn
+        mr_in
+        ms_my
+        ms_bn
         nb_no
+        nl_nl
+        nl_be
         nn_no
         no_no
+        pa_in
         pl_pl
-        pt_br
         pt_pt
-        pa_in
-        ro_md
+        pt_br
         ro_ro
+        ro_md
+        ru_ru
         ru_by
         ru_kz
-        ru_ru
         ru_ua
         sa_in
-        sr_ba
-        sr_rs
-        sr_xk
         sk_sk
         sl_si
-        es_ar
-        es_bo
-        es_cl
-        es_co
-        es_cr
-        es_do
-        es_ec
-        es_sv
-        es_gt
-        es_hn
-        es_mx
-        es_ni
-        es_pa
-        es_py
-        es_pe
-        es_pr
-        es_es
-        es_uy
-        es_ve
-        sw_ke
-        sv_fi
+        sq_al
+        sq_mk
+        sq_xk
+        sr_rs
+        sr_ba
+        sr_xk
         sv_se
+        sv_fi
+        sw_ke
         ta_in
-        tt_ru
         te_in
         th_th
-        tr_cy
         tr_tr
+        tr_cy
+        tt_ru
         uk_ua
-        ur_in
         ur_pk
+        ur_in
         uz_uz
         vi_vn
+        zh_cn
+        zh_hk
+        zh_mo
+        zh_sg
+        zh_tw
       ].freeze
```

Comment:
You could have just done
```suggestion
      ].sort.freeze
```

Reply:
That is helpful for future reference. However, in this case, the sort is not entirely alphabetical as this line https://github.com/soberstadt/adobe-campaign/blob/master/lib/adobe/campaign/language.rb#L16 requires that preferred default language locale be ranked ahead of the other locales for that language, as in https://github.com/soberstadt/adobe-campaign/pull/4/files#diff-0c45cc1aff9fe53797c26a49cb5cd989R94.